static/app/views/monitors/components/overviewTimeline/checkInTimeline.tsx

```diff
@@ -67,6 +72,43 @@ export function CheckInTimeline(props: CheckInTimelineProps) {
   );
 }
 
+export interface MockCheckInTimelineProps extends TimelineProps {
+  mockTimestamps: Date[];
+}
+
+export function MockCheckInTimeline(props: MockCheckInTimelineProps) {
+  const {mockTimestamps, start, end, timeWindowConfig, width} = props;
+
+  const elapsedMs = end.getTime() - start.getTime();
+  const msPerPixel = elapsedMs / width;
+
+  return (
+    <TimelineContainer>
+      {mockTimestamps.map(ts => {
+        const timestampMs = ts.getTime();
+        const left = getBucketedCheckInsPosition(timestampMs, start, msPerPixel);
+
+        return (
+          <Tooltip
+            key={left}
+            title={
+              <DateTime date={timestampMs} format={timeWindowConfig.dateLabelFormat} />
+            }
+            skipWrapper
+          >
+            <JobTick
+              style={{left}}
+              status={CheckInStatus.IN_PROGRESS}
+              roundedLeft
+              roundedRight
+            />
```

Comment:
We'll always make sure they don't touch right?

Reply:
yea they shouldn't touch unless we specifically want it too based on the sample data we request